config/alienwhitelist.txt

```diff
@@ -43,6 +43,7 @@ silvertalismen - Diona
 silvertalismen - Vox
 silvertalismen - Xenochimera
 singo - Gutter
+tastypred - Shadekin Empathy
```

Comment:
You need to set the whitelist for Black-Eyed Shadekin

Shadekin empathy is something completely different.